README.md

```diff
@@ -1,6 +1,5 @@
 [![Version](https://img.shields.io/badge/version-development-orange.svg)](https://github.com/0xProject/0x-mesh/releases)
 [![Docs](https://img.shields.io/badge/docs-website-yellow.svg)](https://0x-org.gitbook.io/mesh)
-[![GoDoc](https://godoc.org/github.com/0xProject/0x-mesh?status.svg)](https://godoc.org/github.com/0xProject/0x-mesh)
```

Comment:
The root GoDoc page doesn't contain anything useful for 99% of developers interested in Mesh. What they might want to see is the Go RPC client and that is already linked elsewhere. I think it makes more sense to direct people to https://0x-org.gitbook.io/mesh as the main place for documentation for Mesh because it has a lot more useful information. What do you think?

Reply:
Yeah I agree. I still think we should surface the Godoc docs somewhere though. Maybe we can link to it from the docs sidebar?